tests/test_n2v.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import unittest
+import networkx as nx
+import numpy as np
+import graspologic.embed.n2v as n2v
+from graspologic.embed.n2v import _Node2VecGraph
+
+
+class Node2VecEmbedTest(unittest.TestCase):
```

Comment:
feels like there should be at least one test that the output is sensible, not just the right shape

Reply:
also same comment about tests for the values of input parameters and error raising which is something we usually do

Reply:
I agree - I'm going to spawn a list of issues out of this for enhancements to it, and this will be one of them.  I don't know how I feel about putting an edge list in the unit test file vs. a test file - there's a lot to like about it, but it's new to me and I'm scared of new things, I guess.

Reply:
https://github.com/microsoft/graspologic/issues/557 @Nyecarr